src/component/TodoInput.jsx

```diff
@@ -1,27 +1,35 @@
-import { useState } from "react";
+import { array, func, string } from "prop-types"
 
-export default function TodoInput({ todos, setTodos, setflag ,task,setTask }) {
-  
+export default function TodoInput({ todos, setTodos, todo, setTodo }) {
+    const handleSubmit = (e) => {
```

Comment:
Instead of having two ways to handle a form submit, we now have one way (HTML form submit).
Ref: https://www.epicreact.dev/how-to-type-a-react-form-on-submit-handler